docs/source/common_issues.rst

```diff
@@ -505,11 +505,19 @@ to see the types of all local variables at once. Example:
    #     b: builtins.str
 .. note::
 
-   ``reveal_type`` and ``reveal_locals`` are only understood by mypy and
-   don't exist in Python. If you try to run your program, you'll have to
-   remove any ``reveal_type`` and ``reveal_locals`` calls before you can
-   run your code. Both are always available and you don't need to import
-   them.
+   ``reveal_type`` and ``reveal_locals`` are understood by mypy during typechecking,
+   and don't have to be imported as functions. However,
+   if you don't import them, then they don't exist at runtime! Therefore,
+   you'll have to remove any ``reveal_type`` and ``reveal_locals`` calls from your program
+   or else Python will give you an error at runtime about those names being undefined.
```

Comment:
The ragged right side makes me unhappy :-(

Reply:
The test it replaces was exceptionally non-ragged, as well! What a shame.

Reply:
Anyway, I deliberately broke it up by clause, (like eg https://sembr.org/ ) which leads to exceptional raggedness. But theoretically will make diffs easier to look at when someone edits this in the future.

Reply:
Hopefully this now sparks joy 😉